src/OrchardCore/OrchardCore.Infrastructure/Documents/VolatileDocumentManager.cs

```diff
@@ -116,7 +116,7 @@ private sealed class UpdateDelegates
         {
             public UpdateDelegate UpdateDelegateAsync;
             public AfterUpdateDelegate AfterUpdateDelegateAsync;
-            public HashSet<object> Targets = [];
+            public HashSet<object> Targets { get; set; } = [];
```

Comment:
Why do you need this change?

```suggestion
            public HashSet<object> Targets { get; init; } = [];
```